### PR TITLE
minor fix on System.Globalization.Calendars

### PR DESCRIPTION
**Pull request by venkat-raman251**
changes for #1921 
Also added the library to Readme.md. @stephentoub could you please have a quick look.

### CODE REVIEW

File: src/System.Globalization.Calendars/tests/project.lock.json [hunk 1]
```diff
@@ -0,0 +1,588 @@
+{
+  "locked": false,
```
Comment (stephentoub) [inline src/System.Globalization.Calendars/tests/project.lock.json]: @ericstj, I've seen this be true elsewhere... should it be changed from false here, too?

**Comment by stephentoub** [comment]
One comment/question; otherwise LGTM.

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by venkat-raman251** [comment]
Do I wait for @ericstj  before I merge?

**Comment by stephentoub** [comment]
Well, let's go ahead I guess.  The vast majority of the files have "locked": true, but I see a few with "locked": false, so we may just need to do a sweep through to clean them all up.

**Comment by venkat-raman251** [comment]
Thanks @stephentoub

**Pull request merged**